### ISSUE TITLE
Icon issue 

### ISSUE THREAD

**Issue by rbouikila** [diogocavilha/fancy-git#24]
Hello,

When i install fancy-git on my linux mint desktop. Icons do not display correctly

![capture du 2018-12-02 22-40-36](https://user-images.githubusercontent.com/4258294/49345309-4f840180-f683-11e8-8007-adadc51da279.png)
.

**Comment by diogocavilha** [comment]
Hello, @rbouikila 

First of all, thank you for installing it.

It seems you have to change your terminal font.
Set the `SourceCodePro` font on your terminal and I think it will work.

Let me know if it works.

**Comment by rbouikila** [comment]
Yes, i install SourceCodePro font but when i open terminal i have same issue.

**Comment by diogocavilha** [comment]
Have you selected that font to be your terminal default font?

**Comment by rbouikila** [comment]
I don't see policy to my terminal, I find solution to Linux Mint Forum.
It's a problem on my computer !!!

**Comment by rbouikila** [comment]
I use by default gnome3-terminal. I install other terminal and it work fine !
I close this issue.

**Comment by diogocavilha** [comment]
Ohh! Right!

Contact me at any problem you have with fancy git. ;)

**Comment by diogocavilha** [comment]
> I use by default gnome3-terminal. I install other terminal and it work fine !
> I close this issue.

Ohh! Right!

Contact me at any problem you have with fancy git. ;)
